include/openhmd.h

```diff
@@ -61,6 +61,8 @@ typedef enum {
 typedef enum {
 	OHMD_GLSL_DISTORTION_VERT_SRC = 0,
 	OHMD_GLSL_DISTORTION_FRAG_SRC = 1,
+	OHMD_GLSL_DISTORTION_VERT_OGL3_SRC = 2,
+	OHMD_GLSL_DISTORTION_FRAG_OGL3_SRC = 3,
```

Comment:
Should probably be called `OHMD_GLSL_330_DISTORTION...` so people know exactly which `#version <number>` header we start the shaders with.